java-manta-benchmark/src/main/java/com/joyent/manta/benchmark/RandomInputStream.java

```diff
@@ -18,6 +18,7 @@
  *
  * @author <a href="https://github.com/dekobon">Elijah Zupancic</a>
  */
+@SuppressWarnings("InputStreamSlowMultibyteRead")
```

Comment:
Since this is in a benchmark, shouldn't we resolve the possible performance problem to avoid ambiguity?